src/components/play/PlayFillInTheBlanks.tsx

```diff
@@ -21,13 +21,40 @@ import Answers from './fillInTheBlanks/Answers';
 import BlankedText from './fillInTheBlanks/BlankedText';
 import Correction from './fillInTheBlanks/Correction';
 
+type State = { answers: Word[]; words: Word[] };
+
+const isWordCorrect = (w: Word) => w.displayed === w.text;
+
+const resetWrongAnswers = (state: State): State => {
+  const newWords = state.words.map((w) => {
+    const isCorrect = isWordCorrect(w);
+
+    if (w.displayed === undefined) {
+      return w;
+    }
+
+    return {
+      ...w,
+      displayed: isCorrect ? w.displayed : '',
```

Comment:
Mhmmm displayed can be an empty string of undefined ?

Reply:
<img width="289" alt="image" src="https://github.com/user-attachments/assets/7582e6da-490a-4c3b-91e8-a425b2990cf3">

@spaenleh, Regarding to the initial state of the quiz for fill in the blanks, the displayed is an empty string and not undefined. So I used the empty string to be coherent with the initial state.